hack/generate-statik.sh

```diff
@@ -38,11 +38,21 @@ elif ! [ -x "$(command -v ${LICENSES})" ]; then
     popd
 fi
 
+# hack for github.com/vbauerster/mpb begin
```

Comment:
This sounds a little fishy. if the script fails before L52 then file will not be renamed.
Look into deleting this file in  exit trap